### PR TITLE
handling of eth_sendRawTransaction

### PR DESCRIPTION
**Pull request by deacix**
We from CryptoManiacs implemented eth_sendRawTransaction handling.

### CODE REVIEW

File: src/js/relayclient/RelayProvider.js [hunk 1]
```diff
@@ -33,7 +34,29 @@ class RelayProvider {
...
+                const params = ethers.utils.parseTransaction(payload.params[0])
+
+                if (this.relayOptions.verbose) {
+
+                    console.log("calling sendAsync" + JSON.stringify(payload))
+                    console.log('transaction', params)
+                }
+
+                params.gasPrice = Number.parseInt(params.gasPrice.toString()).toString(16)
+                params.value = Number.parseInt(params.value.toString()).toString(16)
```
[the remark this answers is not in the document]
Comment (k06a) [reply src/js/relayclient/RelayProvider.js]: That was the fastest solution at that moment, I think you lib could easily handle signed transaction. We had only 1 day to integrate GSN to Dapp, and we did it.

[the remark this answers is not in the document]
Comment (drortirosh) [reply src/js/relayclient/RelayProvider.js]: fair enough, but we're not going to integrate it into the client.
What we do think is _warn_ the user against using  sendRawTransaction, as it behaves different than what the client might think

[the remark this answers is not in the document]
Comment (k06a) [reply src/js/relayclient/RelayProvider.js]: @drortirosh I understand why our code is not working for you, but not understand why you cant just forward signed transaction to relays?

[the remark this answers is not in the document]
Comment (drortirosh) [reply src/js/relayclient/RelayProvider.js]: We can forward signed transactions (or any other RPC call, for that matter) to the backend provider, but only if you deliberately mark this sender in the `skipSenders` list, using:
 `RelayProvider( web3, { skipSenders:{fromAddr} })`

**Pull request closed without merging**